### PR TITLE
updating email template with new logo

### PR DESCRIPTION
**Pull request by shanbady**
### What are the relevant tickets?

Closes https://github.com/mitodl/hq/issues/5674

### Description (What does it do?)
Updates the logo in the header of subscription emails

Note: unrelated to the emails - this PR also contains a data migration to include a small copy change to one of the units

### Screenshots (if appropriate):

Screenshot 2024-10-02 at 11 08 41 AM

### How can this be tested?
Follow instructions on [this PR](https://github.com/mitodl/mit-open/pull/1619) to send yourself a subscription email and validate how it looks

### CODE REVIEW

**Review by mbertrand — COMMENTED** [review]
Not directly related to this PR, but I noticed the unsubscribe link is missing a "/", I think this might be because I have `MITOPEN_APP_BASE_URL=http://open.odl.local:8062` (no ending slash).  Neither does the same setting on RC or prod.  Otherwise looks great.

File: data_fixtures/migrations/0015_unit_page_copy_updates.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    {
+        "name": "mitpe",
+        "offeror_configuration": {
+            "value_prop": (
+                "MIT Professional Education is a leader in technology and "
+                "engineering education for working professionals pursuing "
+                "career advancement, and organizations seeking to meet modern-day "
+                "challenges by expanding the knowledge and skills of their employees. "
+                "Courses are delivered in a range of formats—in-person (on-campus "
+                "and live online), online, and through hybrid approaches—to "
```
[the remark this answers is not in the document]
Comment (shanbady) [reply data_fixtures/migrations/0015_unit_page_copy_updates.py:15]: Yea - not easily visible on github but its a long dash approaches—to https://rc.learn.mit.edu/c/unit/mitpe/

**Comment by shanbady** [comment]
> Not directly related to this PR, but I noticed the unsubscribe link is missing a "/", I think this might be because I have `MITOPEN_APP_BASE_URL=http://open.odl.local:8062` (no ending slash). Neither does the same setting on RC or prod. Otherwise looks great.

good catch! pushed the fix

**Review by mbertrand — APPROVED** [review]
👍

**Pull request merged**